src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -1480,11 +1494,11 @@ subroutine get_obs_pred(                                       &
          N_catlH, tile_data_lH=tile_data_lH )    
     
     ! read out sfmc, rzmc, etc. from tile_data_lH    
-    
+    ! addded asnow option at here
     call get_obs_pred_comm_helper( N_catlH, N_ens, N_TbuniqFreqAngRTMid,         &
-         get_sfmc_lH, get_rzmc_lH, get_tsurf_lH, get_FT_lH, get_Tb_lH, N_fields, &
+         get_sfmc_lH, get_rzmc_lH, get_tsurf_lH, get_FT_lH, get_asnow_lH, get_Tb_lH, N_fields, &
          option=2, tile_data=tile_data_lH,                                       &
-         sfmc=sfmc_lH, rzmc=rzmc_lH, tsurf=tsurf_lH, FT=FT_l, stemp=stemp_lH,    &
+         sfmc=sfmc_lH, rzmc=rzmc_lH, tsurf=tsurf_lH, asnow=asnow_l, FT=FT_l, stemp=stemp_lH,    &
```

Comment:
I think this should read "asnow=asnow_lH" because "tile_data" is unbundled into the "_lH" arrays ("lH" is local with halo).  The same probably goes for "FT=FT_lH", which seems to be a bug in the current develop branch.  (Be careful relying on the "FT" implementation because it was never tested.)